pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -622,7 +622,12 @@ func (nc *Controller) doNoExecuteTaintingPass(ctx context.Context) {
 				// retry in 50 millisecond
 				return false, 50 * time.Millisecond
 			}
-			_, condition := controllerutil.GetNodeCondition(&node.Status, v1.NodeReady)
+			condition := controllerutil.GetNodeCondition(&node.Status, v1.NodeReady)
+			if condition == nil {
```

Comment:
```suggestion
			if condition := controllerutil.GetNodeCondition(&node.Status, v1.NodeReady); condition == nil {
```

Reply:
This variable needs to be referenced below.
https://github.com/kubernetes/kubernetes/blob/3aa8c59fec0bf339e67ca80ea7905c817baeca85/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L629